src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -742,6 +742,9 @@ type RentalVehicle implements Node & PlaceInterface {
 
     """The type of the rental vehicle (scooter, bicycle, car...)"""
     vehicleType: RentalVehicleType
+
+    """The rental vehicle operator's system URL."""
```

Comment:
System URL sounds like a weirdly technical term. The GBFS documentation describes this "The URL of the vehicle share system.", maybe calling it vehicle share system is a bit less vague.

Also, since the system_information.json file contains other things we might expose at some point, maybe we should create a system type and put the url there.

Reply:
Fixed by using the RentalVehicleSystemType instead.